packages/cli/internal/pkg/aws/cdk/execute_cdk_test.go

```diff
@@ -114,3 +160,10 @@ func TestHelperProcess(t *testing.T) {
 		os.Exit(1)
 	}
 }
+
+func waitForChanToClos(channel ProgressStream) {
```

Comment:
nit: `waitForChanToClos` -> `waitForChanToClose`

Reply:
Oh. Thank you for catching that!